xacro_live/xacro_update_handler.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Open Rise Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy.logging as roslog
+from watchdog.events import EVENT_TYPE_MODIFIED
+from watchdog.events import FileSystemEventHandler
+import xacro
+
+from .robot_description_client import RobotDescriptionClient
+from .xacro_observer import XacroObserver
+
+
+class XacroUpdateHandler(FileSystemEventHandler):
+
+    def __init__(self, xacro_observer: XacroObserver, client: RobotDescriptionClient):
```

Comment:
I think that the usage of mutable object as default initializer should be avoided as discussed [here](https://google.github.io/styleguide/pyguide.html#212-default-argument-values)

Reply:
Note I'm not passing any default argument, I'm only specifying type hints.

Reply:
My bad, sorry for the noise